# Folder-4
# Activity-Identify-an-Unknown-Biological-Sequence-Using-BLAST
Tool used : NCBI BLAST 

# 1. Activity Report: Sequence 1 Analysis
# STEP:1 Observe the given sequence

 Observation: The sequence contains only the letters A, T, G, C.
 
 Prediction: This is a DNA sequence.
 
# STEP:2 & 3 BLAST setup 
 
  Tool Used: BLASTn (Nucleotide BLAST) because the input is DNA.

  Database: Standard Nucleotide Collection (nr/nt).

 # STEP:4 Identify the best match( stimulated results ) 

   Gene Name: Zaire ebolavirus isolate Mayinga, complete genome (or Nucleoprotein match).

  Organism: Zaire ebolavirus (Ebola virus)

   Database Used: GenBank / NCBI Nucleotide

   % Identity: 100% (Exact match)

   E-value: 0.0 (Indicates a statistically significant match, not random).
   <img width="1920" height="1080" alt="Screenshot (97)" src="https://github.com/user-attachments/assets/84474096-3424-4954-83f0-73f9cfc137ad" />
   <img width="1920" height="1080" alt="Screenshot (98)" src="https://github.com/user-attachments/assets/f1181684-fca6-44f6-93ae-b77080774348" />
# STEP:5 Verify using alignment 

 Does the alignment support this? Yes.
 
  Reason: The sequence aligns perfectly with the beginning of the Ebola virus genome (specifically the leader region and the start of the Nucleoprotein gene) with no gaps or mismatches.
# STEP 6: Identify the Biological Function:

 Biological Role: This sequence corresponds to the Leader sequence and the start of the Nucleoprotein (NP) gene.

 Function:

 Structural: The Nucleoprotein (NP) encapsulates the viral RNA genome to form the nucleocapsid.

 Disease: It is the causative agent of Ebola Virus Disease (EVD), a severe hemorrhagic fever in humans.
# STEP 7: Classification & Interpretation: 

   Sequence Classification: Gene/Genome Fragment (Viral DNA/RNA form).

   Organism Type: Viral (It is neither prokaryotic nor eukaryotic, though it infects eukaryotic hosts).

   Conservation: This specific sequence is highly conserved among Zaire ebolavirus isolates.
<img width="1920" height="1080" alt="Screenshot (99)" src="https://github.com/user-attachments/assets/10deb7e7-ac3b-4148-a49c-69e79ab73cc3" />
# STEP 8: Final Conclusion:
"The given unknown sequence is identified as a genomic fragment from Zaire ebolavirus and an E-value of 0.0, indicating a strong match. This organism is responsible for severe hemorrhagic fevers."
# INTERPRETATION:
"I identified Sequence 1 as a DNA segment from the Zaire ebolavirus (commonly known as Ebola). The BLAST analysis gave an E-value of 0.0, which indicates a statistically significant match and confirms this is not a random error.

Biologically, this sequence matches the Nucleoprotein (NP) gene. This gene is critical because it helps the virus package its genetic material so it can survive and replicate. This identification confirms that the sample belongs to a dangerous pathogen that causes severe disease in humans."

# 2. Activity Report: Sequence 2 Analysis
# Step 1: Observe the Given Sequence:

  Observation: The sequence contains only the letters A, T, G, C.

  Prediction: This is a DNA sequence.
# Step 2 & 3: BLAST Setup:

  Tool Used: BLASTn (Nucleotide BLAST).

  Database: Standard Nucleotide Collection (nr/nt).
# Step 4: Identify the Best Match (Simulated Results):

  Gene Name: Homo sapiens insulin (INS) gene.

  Organism: Homo sapiens (Human).

  Database Used: GenBank / NCBI Nucleotide.

  % Identity: 100% (Exact match).

  E-value: 0.0 (Indicates a statistically significant match).
  <img width="1920" height="1080" alt="Screenshot (100)" src="https://github.com/user-attachments/assets/c2d60436-74ad-4c3c-b218-47d5b2ef689b" />
  <img width="1920" height="1080" alt="Screenshot (101)" src="https://github.com/user-attachments/assets/569ec63b-4405-4d32-b4b3-6345002a4679" />
# Step 5: Verify Using Alignment:

 Does the alignment support this? Yes.

 Reason: The sequence aligns perfectly with the 5' flanking region (upstream promoter region) of the human insulin gene, which is known for having variable tandem repeats.
# Step 6: Identify the Biological Function:

  Biological Role: This gene codes for the precursor of Insulin (preproinsulin).

Function:

  Regulation: Insulin is a hormone that regulates blood glucose levels by facilitating cellular glucose uptake.

  Disease: Mutations or regulation issues in this gene are directly associated with Diabetes Mellitus.
# Step 7: Classification & Interpretation:

  Sequence Classification: Gene/Genome Fragment (Eukaryotic DNA).

  Organism Type: Eukaryotic (Human).

  Conservation: Insulin is highly conserved across vertebrates, though this specific upstream region contains repeats that can vary between individuals (polymorphic).
  <img width="1920" height="1080" alt="Screenshot (102)" src="https://github.com/user-attachments/assets/a674ebab-ea85-4d25-aa87-8578fa17f92e" />
# step 8: Final Conclusion:

  "The given unknown sequence is identified as the Insulin (INS) gene (specifically the 5' flanking region) from Homo sapiens and an E-value of 0.0, indicating a strong match."
# INTERPRETATION:
"I identified Sequence 2 as a DNA segment from the Human Insulin (INS) gene. The BLAST result gave an E-value of 0.0, which confirms this is a perfect match and definitely not a random error. Biologically, this gene is extremely important because it gives instructions to make insulin, the hormone that controls our blood sugar levels. If this gene has a mutation or doesn't work correctly, it can lead to diabetes. Specifically, the sequence I tested matches the 'regulatory region' at the very beginning of the gene, which acts like a switch to control how much insulin is made."
# 3. Activity Report: Sequence 3 Analysis
# STEP 1: Observe the Given Sequence:

 Observation: The sequence contains letters like M, L, K, F, W, E (Amino Acids). It does not contain only A, T, G, C.

 Prediction: This is a Protein sequence.
# STEP 2 & 3: BLAST Setup:

 Tool Used: BLASTp (Protein BLAST).

 Database: Non-redundant protein sequences (nr).
# STEP 4: Identify the Best Match:

 Gene/Protein Name: Envelope Glycoprotein gp160 / gp41 (Transmembrane subunit).

 Organism: Human immunodeficiency virus 1 (HIV-1).

 Database Used: GenBank / NCBI Protein.

 % Identity: 100% (Exact match to HIV-1 isolates).

 E-value: 0.0 (Strong, statistically significant match).
 <img width="1920" height="1080" alt="Screenshot (103)" src="https://github.com/user-attachments/assets/742bc51a-04ea-4d57-8242-7c5391d3f1ab" />
 <img width="1920" height="1080" alt="Screenshot (104)" src="https://github.com/user-attachments/assets/520b0cd7-8714-4535-aa03-6eacaf56014a" />
 # STEP 5: Verify Using Alignment:
 Does the alignment support this? Yes.

   Reason: The sequence matches the Membrane Proximal External Region (MPER) and the Transmembrane domain of the HIV-1 gp41 protein. This region is highly characteristic of the virus.
# STEP:6 Identify the biological function
Biological Role: This protein acts as the "key" for the virus to enter human cells.

 Function:

   Structural/Mechanical: It mediates the fusion of the viral membrane with the host cell membrane, allowing the virus to inject its genetic material.

 Disease: It is the causative agent of Acquired Immunodeficiency Syndrome (AIDS).
# STEP 7: Classification & Interpretation: 
  Sequence Classification: Viral Protein.

  Organism Type: Retrovirus (Lentivirus).

  Conservation: The "MPER" region (found in this sequence) is one of the most conserved parts of the HIV envelope, often targeted by vaccine researchers.
<img width="1920" height="1080" alt="Screenshot (105)" src="https://github.com/user-attachments/assets/d66dec47-479b-4709-8070-daaba3a7a116" />
# STEP : 8 Final Conculsion 
  "The given unknown sequence is identified as the Envelope Glycoprotein (gp41) from HIV-1 and an E-value of 0.0, indicating a strong match."
# INTERPRETATION 
"I identified Sequence 3 as a Protein sequence from the HIV-1 virus (the virus that causes AIDS). The BLAST result gave an E-value of 0.0, which means the match is certain.

Specifically, this sequence is part of the gp41 envelope protein. Biologically, this protein acts like a grappling hookâ€”it helps the virus fuse with human immune cells so it can get inside and infect them. Because this specific part of the protein doesn't change much between different strains of the virus, scientists often study it to try and create vaccines."





   

   
